src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -694,7 +698,7 @@ public static bool IsBaseInterface(TypeSymbol baseType, TypeSymbol derivedType,
 
             foreach (var iface in d.AllInterfacesWithDefinitionUseSiteDiagnostics(ref useSiteDiagnostics))
             {
-                if (HasIdentityConversion(iface, baseType))
+                if (HasIdentityConversionInternal(iface, baseType, includeNullability: false))
```

Comment:
>includeNullability: false [](start = 67, length = 25)

It would be good to add a comment explaining why it is the right thing to do to always pass false here.